webpack/assets/javascripts/react_app/components/common/Date/AbsoluteDate.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedDate } from 'react-intl';
+
+class AbsoluteDate extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    let date, title;
+
+    if (this.props.date) {
+      date = this.props.date;
+      title = this.props.intl.formatRelative(date);
+
+      return (
+        <span title={title}>
+          <FormattedDate value={date}
+            day='2-digit'
+            month='short'
+            hour='2-digit'
+            minute='2-digit'
+            hour12={false} />
```

Comment:
I know that most of the world uses 24hr time but to set this to false is saying that we are okay with actively showing US users a time that they may not readily understand.

Shouldn't this be updated by whatever intl library we are using based on the user's locale?